src/notation/view/percussionpanel/percussionpanelpadlistmodel.cpp

```diff
@@ -99,61 +116,76 @@ void PercussionPanelPadListModel::endDrag(int endIndex)
     m_dragStartIndex = -1;
 }
 
-void PercussionPanelPadListModel::setDrumset(const mu::engraving::Drumset* drumset)
+void PercussionPanelPadListModel::setDrumset(const engraving::Drumset* drumset)
```

Comment:
Can we pass it by reference and create a copy instead of using pointers?

Reply:
I prefer to use a pointer purely because, conceptually, the percussion panel won't always have a drumset. In this case it's really convenient to be able to null check the stored drumset.